.github/workflows/terraform-checks.yml

```diff
@@ -67,21 +67,22 @@ jobs:
           curl -sL "https://github.com/$(curl -sL https://github.com/aquasecurity/tfsec/releases/latest | grep -o -E "/.+?_linux_amd64.tar.gz")" > tfsec.tar.gz &&
           tar -xf tfsec.tar.gz tfsec && rm tfsec.tar.gz && install tfsec /usr/local/bin && rm tfsec
       - name: Run tfsec scan
-        uses: pre-commit/action@v2.0.3
+        uses: pre-commit/action@v3.0.0
         with:
           extra_args: terraform_tfsec --all-files
-  terrascan_job:
-    runs-on: ubuntu-latest
-    name: Run terrascan
-    steps:
-      - name: Checkout repo
-        uses: actions/checkout@master
-      - uses: actions/setup-python@v3
-      - name: Install terrascan
-        run: >-
-          curl -sL "$(curl -sL https://api.github.com/repos/accurics/terrascan/releases/latest | grep -o -E "https://.+?_Linux_x86_64.tar.gz")" > terrascan.tar.gz &&
-          tar -xf terrascan.tar.gz terrascan && rm terrascan.tar.gz && install terrascan /usr/local/bin && rm terrascan
-      - name: Run Terrascan
-        uses: pre-commit/action@v2.0.3
-        with:
-          extra_args: terrascan --all-files
+  ## Merge back when terrascan updates upstream
+  # terrascan:
+  #   runs-on: ubuntu-latest
+  #   name: Run terrascan
+  #   steps:
+  #     - name: Checkout repo
+  #       uses: actions/checkout@master
+  #     - uses: actions/setup-python@v3
+  #     - name: Install terrascan
+  #       run: >-
+  #         curl -sL "$(curl -sL https://api.github.com/repos/accurics/terrascan/releases/latest | grep -o -E "https://.+?_Linux_x86_64.tar.gz")" > terrascan.tar.gz &&
+  #         tar -xf terrascan.tar.gz terrascan && rm terrascan.tar.gz && install terrascan /usr/local/bin && rm terrascan
+  #     - name: Run Terrascan
+  #       uses: pre-commit/action@v3.0.0
+  #       with:
+  #         extra_args: terrascan --all-files
```

Comment:
what are we waiting on?

Reply:
some 1.2 features are not supported yet, causing the scan to fail

Reply:
do we have an issue to track re-enabling?